manta-pay/src/simulation/mod.rs

```diff
@@ -72,7 +72,6 @@ where
         utxo_accumulator_model,
     );
     signer.load_accounts(AccountTable::new(KeySecret::sample(rng)));
-    signer.update_authorization_context();
```

Comment:
Why was this removed?

Reply:
`load_accounts` now calls `update_authorization_context` internally.